db/seeds.rb

```diff
@@ -60,9 +61,13 @@
                  collect_starts_at: 10.day.from_now,
                  collect_ends_at: 20.days.from_now)
 
-    5.times do |index|
-      Purchase.create(user: User.find(index+1), offer: Offer.last, amount: 1, status: PurchaseStatus::CONFIRMED,
+    5.times do |index2|
+      user  = User.find(index2+1)
+      offer = Offer.last
+      OldPurchase.create(user: user, offer: offer, amount: 1, status: OldPurchaseStatus::CONFIRMED,
```

Comment:
I'll remove everything related to `OldPurchase` when we define the final scope for this new payment gateway implementation. But probably I'll remove this after the merge, making another PR.